client/src/controllers/PreviewController.test.js

```diff
@@ -221,10 +221,11 @@ describe('PreviewController', () => {
     expect(newIframe.id).toEqual(oldIframeId);
     expect(newIframe.src).toEqual(expectedUrl);
     expect(newIframe.getAttribute('style')).toBeNull();
-    expect(newIframe.contentWindow.scroll).toHaveBeenCalledWith(
-      oldIframe.contentWindow.scrollX,
-      oldIframe.contentWindow.scrollY,
-    );
+    expect(newIframe.contentWindow.scroll).toHaveBeenCalledWith({
+      top: oldIframe.contentWindow.scrollX,
+      left: oldIframe.contentWindow.scrollY,
```

Comment:
Oops this should've been
```suggestion
      top: oldIframe.contentWindow.scrollY,
      left: oldIframe.contentWindow.scrollX,
```

(and all the rest of the tests) 🤦 . They pass because the values are both `0`